src/azure-cli-core/azure/cli/core/aaz/_field_value.py

```diff
@@ -182,8 +182,7 @@ def __len__(self):
         return len(self._data)
 
     def __iter__(self):
-        for key in self._data:
-            yield key
+        yield from self._data
```

Comment:
`src/azure-cli-core/azure/cli/core/aaz/_field_value.py:185:8: R1737: Use 'yield from' directly instead of yielding each element one by one (use-yield-from)`